scripts/congrats-bot.mjs

```diff
@@ -9,7 +9,7 @@ setDiscordMessage(COMMIT_AUTHOR, COMMIT_ID, COMMIT_MESSAGE);
  * @param {string} commitMsg A full commit message
  */
 function setDiscordMessage(author, id, commitMsg) {
-	const commitMessage = commitMsg.split('\n').shift().replaceAll('`', '');
+	const commitMessage = commitMsg.split('\n').shift().replaceAll('`', '').replaceAll('--', '––');
```

Comment:
Interesting. Didn’t think of this approach!

I think we can even go one better:

```suggestion
	const commitMessage = commitMsg.split('\n').shift().replaceAll('`', '').replaceAll('--', '-​-');
```

Very hard to see, but I’ve added a zero-width space between the two hyphens. What do you think?

Reply:
Sounds like an even smoother approach! Added it!
The only worry is discord might not read it as a character, but from my test, it seems to.